crates/topos-api/proto/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@topos-protocol/topos-grpc-client-stub",
-  "version": "0.1.6",
+  "version": "0.1.7",
```

Comment:
So this is a manual thing we have to do for each .proto update? Are we using semver here?

Reply:
Yes but we could remove the whole package if I understood correctly @sebastiendan ?

Reply:
Yes, we can remove that npm package as a whole. It was initially used to build js stubs when the idea was to use grpc as the TCE public API, but then we realized grpc wasn't made for that purpose and we switched to graphql.

@Freyskeyd Is it planned in the Protocol team to remove the package, or should I open a PR?